src/esm_runscripts/filelists.py

```diff
@@ -892,7 +892,7 @@ def check_for_unknown_files(config):
     return config
 
 
-def resolve_symlinks(file_source):
+def resolve_symlinks(config, file_source):
```

Comment:
How did this ever work in the first place? config is never defined in the local scope???